### PR TITLE
Activate cloud plugins explicitely

### PR DESCRIPTION
**Pull request by rdettai**
Change described in https://app.shortcut.com/tenzir/story/36533/fix-the-way-cloud-plugins-are-loaded

### :memo: Reviewer Checklist

Review this pull request by ensuring the following items:

- [ ] All user-facing changes have changelog entries
- [x] User-facing changes are reflected on [vast.io](https://github.com/tenzir/vast/tree/master/web)

### CODE REVIEW

**Comment by rdettai** [comment]
Starts off https://github.com/tenzir/vast/pull/2473, so requires it to be merged first

**Comment by rdettai** [comment]
https://github.com/tenzir/vast/actions/runs/2883267769

File: cloud/aws/cli/main.py [hunk 1]
```diff
@@ -25,21 +32,34 @@ def print_help(self) -> None:
...
   -h [STRING], --help[=STRING]   Show core or per-task help and exit.
-  -V, --version                  Show version and exit.      
+  -V, --version                  Show version and exit.
+
+Plugins:
 """
         )
+        active = common.active_plugins()
+        plugin_entry = lambda name: f"[{'x' if name in active else ' '}] {name}"
+        self.print_columns([(plugin_entry(p[0]), p[1].__doc__) for p in list_plugins()])
```
Comment (rdettai) [inline cloud/aws/cli/main.py:42]: With just this tiny tweak, this PR now also solves [35748](https://app.shortcut.com/tenzir/story/35748/vast-cloud-plugin-level-doc)

**Comment by rdettai** [comment]
https://github.com/tenzir/vast/actions/runs/2917862379

**Comment by KaanSK** [comment]
Apart from this review, we can discuss about many readme.MD and other documentation files under different folders which makes it really hard to follow. We can just have 1 documentation file including everything.

Additionally, the .env.example template should have following keys as well. These are referenced in aws-pro and can cause confusion.

```
VAST_CLOUD_PLUGINS=
VAST_VERSION=
```

**Review by KaanSK — CHANGES_REQUESTED** [review]
I was able to initialize and destroy pro environment. The issue with environment variables documentation will be followed in another story.

I commented on changelog entry for a typo.

**Comment by rdettai** [comment]
@KaanSK I addressed you request and created a separate internally tracked [issue](https://app.shortcut.com/tenzir/story/36858/create-a-list-of-all-environment-variables) for env variable template.

File: cloud/aws/cli/main.py (hunk 1 above)
Comment (KaanSK) [reply cloud/aws/cli/main.py:42]: LGTM

**Review by KaanSK — APPROVED** [review]
